.github/workflows/release.yml

```diff
@@ -139,7 +155,7 @@ jobs:
       - name: Package node-for-redis chart
         env:
           OPERATOR_VERSION: ${{ needs.operator.outputs.version }}
-        run: helm package charts/node-for-redis --version=$OPERATOR_VERSION --app-version=${{ env.REDIS_VERSION }} --destination helm-releases
+        run: helm package charts/node-for-redis --version=$OPERATOR_VERSION --app-version=${{ env.REDIS_VERSION_6 }}-${{ env.REDIS_VERSION_7 }} --destination helm-releases
```

Comment:
`appVersion`. should it have both Redis versions? Not sure if the linter will like that...

We don't want to create separate versions of the chart either. The may impact how we tackle https://github.com/IBM/operator-for-redis-cluster/blob/main/charts/node-for-redis/templates/rediscluster.yaml#L73 as well.

Reply:
> Not sure if the linter will like that...

Do you want to approve the tests to check how the linter behaves?

Reply:
Done